### ISSUE TITLE
Bring up to date with Flixel-Community

### ISSUE THREAD

**Issue by greysondn** [FlixelCommunity/Flixel-Power-Tools#24]
This is one of those big projects. I'm happy to contribute the snatches I may translate over soon enough, but I think at least a dev branch in the main repo or something similar would be "nice" before attempting to do so.

**Comment by Dovyski** [comment]
That is great! Thanks for the interesest on keeping Flixel Power Tools updated.

A `dev` branch seems like a nice idea to me. Inspecting FPT branches, I've noticed the `2.0` one though. It contains some structural changes that @photonstorm was working on. Would you like me to create the `dev` branch off of `master` or `2.0`?

**Comment by greysondn** [comment]
https://github.com/FlixelCommunity/Flixel-Power-Tools/compare/photonstorm:2.0...master

Given the diff, my sincere opinion is that throwing away a decent hunk of work (especially where it's not something I'm likely to touch) is bad. Can we vet it into `master`?

Otherwise `dev` comes off `2.0` and can rejoin master at such time as it can be vetted to match at least `flixel-community 3.0` as a static target (I am only one man. I'd love to chase a moving target, but...)

I am willing to fix problems that I am able to see and/or fix as I work with the Power-Tools, basically committing fixes I make upstream... however, I must point out that most of my work has been in refactoring and hand-inserting patches and this is not likely to be different. Most of my changes would be simply to update the API to match what I understand as the next current milestone for `Flixel-Community`.

It may be in our interests to try to get the outstanding pull requests rebased, too, but I'll leave that at maintainer's discretion.

**Comment by Dovyski** [comment]
I agree with you, that approach is perfect! Do you think you can manage to do it all by issuing pull requests?

**Comment by greysondn** [comment]
I am quite alright with issuing my work as pulls. I'm not sure what else you may be asking of me here.

**Comment by Dovyski** [comment]
I am not asking anything else :) You are good to go. We will discuss/review the PR's (if needed) as they arrive.

**Comment by greysondn** [comment]
I sat down to tackle a piece of this for this morning. Yet, there is no `dev` branch.

Clearly something is misunderstood here, though "who" and "what" is likewise not clear to me.

**Comment by Dovyski** [comment]
I'm sorry about that! I've just created the branch. I think you should be able to work on this now.

**Comment by greysondn** [comment]
As I mentioned in [Flixel-Community Iss. 227](https://github.com/FlixelCommunity/flixel/issues/227), I'll be working against [7232E06F](https://github.com/FlixelCommunity/flixel/tree/7232e06ff9536aba45c91061eea5b1e5371b9570)

**Comment by Dovyski** [comment]
Ok, go ahead.

**Comment by greysondn** [comment]
Relatively easy question this time. Would you prefer that I leave classes where they are or move them someplace more aligned with the current flixel-community hierarchy?

For example, the first class I find myself working on is `FlxBitmapFont` . It has a fully-qualified name of `org.flixel.plugin.photonstorm.FlxBitmapFont`.
- Telling me "yes" means to move it to the current equivalent in plugin namespace for flixel-community. Its fully-qualified path name will become `flixel.plugin.photonstorm.FlxBitmapFont` (if that's not agreeable, I'm certainly open to other namespaces other than `photonstorm`.)
- Telling me "no" means to leave it alone. It will retain the fully-qualified name `org.flixel.plugin.photonstorm.FlxBitmapFont`.

There's no hurry on this one - all I'd have to fix, presumably, is my own imports right now after the move - but there may be commits that are just "move file to new location, fix name".

**Comment by Dovyski** [comment]
I think moving them to the current flixel-community hierarchy is the best way to go. Keeping the old `org.flixel` prefix instead of the new `flixel.` one will only cause confusion.
